index.bs

```diff
@@ -4702,15 +4702,15 @@ Registry Definitions</h4>
 			and which is responsible for evaluating whether such requests
 			satisfy the criteria defined in the [=registry definition=].
 
-			The [=custodian=] may be the [=Working Group=], the [=Team=], or a delegated entity.
+			The [=custodian=] may be the initiating [=chartered group|chartered=] or [=elected group=], the [=Team=], or a delegated entity.
```

Comment:
I think we can simplify this to "[=group=]", no?

Reply:
`[=group=]` on its own links to a definition that also includes the AC. I don't think the AC should be in the business of publishing anything